core/src/ctx.rs

```diff
@@ -49,11 +52,19 @@ pub async fn init(api_endpoint: Option<String>, access_token: String) -> Result<
 		return Err(Error::InvalidAgentKind);
 	};
 
+	// Get bootstrap
+	let bootstrap: rivet_api::models::CloudBootstrapResponse =
+		apis::cloud_api::cloud_bootstrap(&openapi_config_cloud)
+			.await
+			.map_err(|source| Error::BootstrapFail { source })?;
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 45 to 59]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Can we drop this in a `tokio::try_join!`?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/rivet-gg/cli/158?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
[![robothello](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/WjlTemxb6oA4PgZFmj08/1eeb7517-02c3-4010-9d69-2169c5e9cf5b/robothello.gif)](https://app.graphite.dev/macros/rivet-gg)